### PR TITLE
ci: fix github action free disk space 

### PR DESCRIPTION
**Pull request by subhamkrai**
**Description of your changes:**
Since `google-cloud-sdk` is renamed with `google-cloud-cli`,
the github action is failing to remove the older name and this
is causing ci issues. Applying changes suggested by community to
manually remove some packages to resolve the issue.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Comment by subhamkrai** [comment]
> commenting here instead of the issue in the cleanup. look closely what I wrote in the issue description:
> 
> ```
> - name: Free Disk Space
>       uses: jlumbroso/free-disk-space@main
>       with:
>         tool-cache: true
>         android: true
>         dotnet: true
>         haskell: true
>         large-packages: false
>         swap-storage: true
> ```
> 
> Notice the `large-packages: false`. You have it as `true` in your build. You need _all_ of the 3 stages I posted there to emulate the:
> 
> * name: Free Disk Space
>   uses: jlumbroso/free-disk-space@main
>   with:
>   tool-cache: true
>   android: true
>   dotnet: true
>   haskell: true
>   large-packages: true
>   swap-storage: true

@wind57 what I understand is I need to `large-packages: false` and let the command remove the large-packages which is effectively makes it `large-packages: true`?

**Comment by sp98** [comment]
add reference to the issue posted in the free-disk-usage repo - https://github.com/jlumbroso/free-disk-space/issues/17

**Comment by subhamkrai** [comment]
> add reference to the issue posted in the free-disk-usage repo - [jlumbroso/free-disk-space#17](https://github.com/jlumbroso/free-disk-space/issues/17)

thanks for the link @sp98 

@travisn do we want to go ahead with this pr or wait to be fixed on the free-disk-space repo?

File: tests/scripts/github-action-helper.sh [hunk 1]
```diff
@@ -679,6 +679,26 @@ function test_csi_nfs_workload {
   kubectl exec -t pod/csinfs-demo-pod -- ls -alh /var/lib/www/html/
 }
 
+function remove_gcloud_in_runner() {
```
Comment (travisn) [inline tests/scripts/github-action-helper.sh:682]: We will be able to remove this workaround in the future?

**Review by travisn — CHANGES_REQUESTED** [review]
Let's go ahead and merge this to get the CI green again. We can have a follow-up PR to remove any workarounds when the fix is available from the other repo.

nit: Please update the commit and PR title to describe the change that we're fixing the free disk space action

File: tests/scripts/github-action-helper.sh (hunk 1 above)
Comment (subhamkrai) [reply tests/scripts/github-action-helper.sh:682]: yes

**Review by travisn — APPROVED** [review]

**Pull request merged**